sdk/inc/azure/core/az_result.h

```diff
@@ -99,15 +98,6 @@ typedef enum
 
   AZ_ERROR_NOT_SUPPORTED = _az_RESULT_MAKE_ERROR(_az_FACILITY_CORE, 7),
 
-  // Platform
-  AZ_ERROR_OUT_OF_MEMORY = _az_RESULT_MAKE_ERROR(
-      _az_FACILITY_PLATFORM,
-      2), ///< Dynamic memory allocation request was not successful.
-
-  AZ_ERROR_HTTP_PLATFORM = _az_RESULT_MAKE_ERROR(
```

Comment:
Please add an entry to the changelog for this change, as it is a breaking change.